flink-runtime/src/main/java/org/apache/flink/streaming/api/connector/sink2/GlobalCommitterOperator.java

```diff
@@ -95,6 +94,7 @@
 public class GlobalCommitterOperator<CommT, GlobalCommT> extends AbstractStreamOperator<Void>
         implements OneInputStreamOperator<CommittableMessage<CommT>, Void> {
 
+    private static final int MAX_RETRIES = 10;
```

Comment:
IMO we should make this a Flink config specific to the sink and allow users to opt-out with the failed committable mechanism. 

Currently the number is duplicated in committer and global committer.